openmc/deplete/operator.py

```diff
@@ -96,17 +98,23 @@ class Operator(TransportOperator):
         Reaction rates from the last operator step.
     burnable_mats : list of str
         All burnable material IDs
+    heavy_metal : float
+        Initial heavy metal inventory
     local_mats : list of str
         All burnable material IDs being managed by a single process
     prev_res : ResultsList
         Results from a previous depletion calculation
+    diff_burnable_mats : bool
+        Whether to differentiate burnable materials with multiple instances
 
     """
-    def __init__(self, geometry, settings, chain_file=None, prev_results=None):
+    def __init__(self, geometry, settings, chain_file=None, prev_results=None,
+                 diff_burnable_mats=True):
```

Comment:
I would prefer this by `False` by default since it effectively modifies the model the user made. Asking to differentiate materials should be a deliberate action.

Reply:
Indeed, to modify the user's model sounds not a good default option, but what if this applies to most cases of depletion simulations? Especially for beginning users, they are more likely to forget to turn on (or even don't aware of) this option though they really want it? I'm not stuck with this but just thought most codes, like mcnp/serpent/RMC, expand the burnable materials by default. If a user wants to keep a mulitple-instance material as one single depletion region, he must also make sure the volume is correct, it sounds more like an operation for advanced users (maybe this is more like a deliberate option?). Let me know your final decision. @paulromano

Reply:
MCNP does not expand burnable materials in lattices/universes by default. Serpent has an option to subdivide a fuel pin into rings, but if you want to treat each instance of a fuel pin as a separate burnable material, you have to manually replicate it in Serpent too. I'm still of the opinion that a user should ask for materials to be differentiated rather than having to ask for them to be grouped together when they already are.

@openmc-dev/technical-committee I'm happy to entertain other opinions on this.

Reply:
Oh sorry, I was misled about the mcnp/serpent usages. I also learned there are more use-cases than what I thought to group the depletion regions for the sake of performance or symmetry, for example, the depletion for massive triso particles. To leave the materials as what they are specified by default seems really reasonable to me now. But I suggest we may later emphasis on how to use this option in the depletion documentation or examples. I'll change it back if there are no other opinions. Thanks @paulromano